tests/system/providers/amazon/aws/example_emr.py

```diff
@@ -143,6 +143,7 @@ def get_step_id(step_ids: list):
     create_job_flow = EmrCreateJobFlowOperator(
         task_id="create_job_flow",
         job_flow_overrides=JOB_FLOW_OVERRIDES,
+        waiter_delay=60
```

Comment:
I think problem with EMR Operators, that with default parameters it raise Deprecation Warning.
But maybe it better to solve it as follow up task. 

cc @vincbeck @o-nikolas @ferruzzi WDYT?

Reply:
Not sure I understand, `waiter_delay` is not deprecated

Reply:
If user do not provide anything than some EMR operators raise `AirflowProviderDeprecationWarning`, which I guess should not be a case.

`waiter_max_attempts` and `waiter_delay` are arguments which replace `waiter_countdown` and `waiter_check_interval_seconds`, however `waiter_max_attempts` and `waiter_delay` has None as default as well as `waiter_countdown`, however deprecated `waiter_check_interval_seconds` has default value which resulted to raise `AirflowProviderDeprecationWarning` warning unless user explicitly not set it to `None`

https://github.com/apache/airflow/blob/830b71b3be66b970340e85cb8f4a297625c71ea1/airflow/providers/amazon/aws/operators/emr.py#L726-L761

Reply:
So maybe instead of fixing system tests, we should fix operators itself.
And if we fix operators, that system tests should be fixed also

Reply:
I agreed, either we need to fix the code or adjust the defaults at the time of removing deprecated code.

Reply:
Makes sense to me as well